packages/terra-action-header/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Unreleased
 * Updated misleading `children` documentation
 * Made required updates to consume terra-toolkit v5 and terra-dev-site v5
 * Adjusted header-icon top styling
+* Changed header-icon css modifiers (back, minimize, etc) to use background-image
```

Comment:
We should add a removed & added section for the changed themeable variables

Reply:
I think I covered this [here](https://github.com/cerner/terra-core/pull/2446/commits/089bba3f41284538b634d4cdf7f35481959c2e11), but let me know if it needs adjustment!